File: iFavcall/iFavcall/ViewController.m
//
//  ViewController.m
//  iFavcall
//
//  Created by Richards Louis on 07/11/12.
//  Copyright (c) 2012 Richards Louis. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

@synthesize address;
@synthesize browser;

- (IBAction) addressEntered:(UITextField *)sender
{
    NSURL *url = [NSURL URLWithString:sender.text];
    NSURLRequest *request = [NSURLRequest requestWithURL:url];
    [self.browser loadRequest:request];
    self.browser.scalesPageToFit = YES;
    [sender resignFirstResponder];
}

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: GluepingmagMac/Glueping Mag/Glueping Mag/ViewController.swift
//
//  ViewController.swift
//  Glueping Mag
//
//  Created by Richards Louis on 14/04/16.
//  Copyright © 2016 Richiesoft inc. All rights reserved.
//

import Cocoa
import WebKit

class ViewController: NSViewController {
    
    @IBOutlet weak var webView: WebView!
    @IBOutlet weak var progressIndicator: NSProgressIndicator!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        let url = Bundle.main.url(forResource: "index", withExtension:"html")
        let request = URLRequest(url: url!)
        self.webView.mainFrame.load(request)
        
         /* let urlString = "http:// website address"
         self.webView.mainFrame.loadRequest(NSURLRequest(URL: NSURL(string: urlString)!)) */

        
        // Do any additional setup after loading the view.
    }
    
    override var representedObject: Any? {
        didSet {
            // Update the view, if already loaded.
        }
    }
    
    

   //PROGRESS INDICATOR FOR WEBSITE LOADING STRAIGHT
 
    func webView(_ sender: WebView!, didStartProvisionalLoadForFrame frame: WebFrame!)
    {
        progressIndicator.startAnimation(self)
    }
    
    func webView(_ sender: WebView!, didFinishLoadForFrame frame: WebFrame!)
    {
        progressIndicator.stopAnimation(self)
    }
    
    
}

